src/palrt/path.cpp

```diff
@@ -280,7 +280,7 @@ STDAPI_(BOOL) PathCanonicalizeW(LPWSTR lpszDst, LPCWSTR lpszSrc)
     BOOL fUNC;
     int cchPC;
 
-    RIPMSG(lpszDst && IS_VALID_WRITE_BUFFER(lpszDst, WCHAR, MAX_PATH), "PathCanonicalize: caller passed bad lpszDst");
+    RIPMSG(lpszDst && IS_VALID_WRITE_BUFFER(lpszDst, WCHAR, MAX_LONGPATH), "PathCanonicalize: caller passed bad lpszDst");
```

Comment:
The spec for PathCanonicalize is hardcoded to use MAX_PATH - https://msdn.microsoft.com/en-us/library/windows/desktop/bb773569(v=vs.85).aspx . We should not be changing to use MAX_LONGPATH for PAL.